include/glow/Graph/Graph.h

```diff
@@ -197,40 +197,40 @@ class Function final : public Named {
   ///@{
 
   ConvolutionNode *createConv(llvm::StringRef name, NodeValue input,
-                              size_t depth, llvm::ArrayRef<size_t> kernels,
-                              llvm::ArrayRef<size_t> strides,
-                              llvm::ArrayRef<size_t> pads, unsigned group);
+                              size_t depth, llvm::ArrayRef<unsigned> kernels,
```

Comment:
Instead of directly using `unsigned`, do we what to use a type alias, like @opti-mix suggested, in case we want to change it in the future?

Reply:
Yeah that probably would have been a good idea!